### PR TITLE
Set OTEL_EXPORTER_OTLP_METRICS_ENDPOINT needed for Java agent

### PR DESCRIPTION
**Pull request by dineshg13**
Fixes a part of #73 

## Changes

Please provide a brief description of the changes here.

1. Update javaagent jar to the latest version
2. Set OTEL_EXPORTER_OTLP_METRICS_ENDPOINT as required by java agent . The variables are listed here https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#otlp-exporter-both-span-and-metric-exporters

Post this change , when we do `docker compose up` , we should see jvm metrics in prometheus

Screen Shot 2022-07-26 at 6 39 06 PM

### CODE REVIEW

**Comment by puckpuck** [comment]
Should we be using the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable only? We only have a single collector endpoint for all OTLP data, and shouldn't need to use the TRACES and METRICS specific flavors.

**Comment by julianocosta89** [comment]
@puckpuck, I'd argue that if we keep `_TRACES` and `_METRICS` we are "showcasing" how to use env vars to configure those 2 signals, even though they are pointing to the same collector.

Personally I'd accept this PR.

**Comment by austinlparker** [comment]
I suppose explicit is better than implicit, although I wonder if there are other implications of using the separate env vars for configuration?

**Comment by dineshg13** [comment]
i agree with @julianocosta89  & @austinlparker  . i like variables being explicit in demo application.

**Review by austinlparker — APPROVED** [review]
This was discussed at the SIG meeting and we'd like to use explicit variables for each signal.

File: .env [hunk 1]
```diff
@@ -9,6 +9,8 @@ COLLECTOR_ZIPKIN_HOST_PORT=:9411
 OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://otelcol:4317
 OTEL_EXPORTER_OTLP_ENDPOINT=${OTEL_EXPORTER_OTLP_TRACES_ENDPOINT}
```
Comment (reyang) [inline .env:11]: Curious, any reason to leave a blank here? Doesn't seem to be consistent with Ln9-10.

**Review by reyang — APPROVED** [review]

**Comment by cartersocha** [comment]
@dineshg13 could you please update the changelog too

File: .env (hunk 1 above)
Comment (dineshg13) [reply .env:11]: no reason. Removing it

**Pull request merged**